packages/portal-proto/src/features/GenomicTables/SomaticMutationsTable/SMTableContainer.tsx

```diff
@@ -77,16 +76,14 @@ export interface SMTableContainerProps {
 
 export const SMTableContainer: React.FC<SMTableContainerProps> = ({
   selectedSurvivalPlot = {},
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  handleSurvivalPlotToggled = (_1: string, _2: string, _3: string) => null,
-
+  handleSurvivalPlotToggled = undefined,
   geneSymbol = undefined,
   projectId = undefined,
   genomicFilters = { mode: "and", root: {} },
   cohortFilters = { mode: "and", root: {} },
   caseFilter = undefined,
-  handleSsmToggled = () => null,
```

Comment:
These were creating a new instance when the component rerendering, causing the react table to rerender each time too.